### ISSUE TITLE
[Core] Remove REST Webservices

### ISSUE THREAD

**Issue by brusch** [pimcore/pimcore#6369]

**Comment by mbolka** [comment]
@brusch Do you have plans to move it to the extension, so REST API can be used with Pimcore 7? Our [Pimcore-Magento Connector](https://github.com/divanteltd/pimcore-magento2-bridge)  is heavily relying on this feature (mapping pimcore object to standardized structure, reading assets)  and the current implementation of GraphQL is not usable for effective data transfer of dynamic product structures (magento need for ex. information about data type)

**Comment by brusch** [comment]
@mbolka there are toughts about providing this as an commercial bundle as part of our enterprise LTS products (also in combination with our import/export sync bundles). But so far the demand is very low, so not really on our list. 
If you're interested, please get in touch with our sales, they can share our offering as well as the pricing.

**Comment by dpfaffenbauer** [comment]
@brusch that is actually a real concern for lots of my customers as well. the Data Hub is great and I also spent time to make it even better. but there is one big disadvantage of the data-hub compared to the rest services: You have to add every single field to the allowed fields, which makes it impossible for some scenarios where the customer himself adds bricks/fields and stuff.

One solution might be: Add a config to DataHub allowing to get the whole object as is, without configuring single fields. This might also resolve @mbolka and Divantes Problem with the Magento Connector.

**Comment by brusch** [comment]
Like I said ... if there's enough interest for a commercial bundle we'd definitely port the functionality and further support it for future versions. 
If you or your clients are interesting in such offering, please just contact our sales 😊

**Comment by brusch** [comment]
A compatibility bundle is now available as part of our Enterprise Subscription: 
https://pimcore.com/en/platform/subscription

**Comment by aarongerig** [comment]
If anyone is interested, we just released an open source bundle for REST interfaces with Pimcore DataHub. Among other things, this gives the possibility to also connect Pimcore with e.g. CI HUB. We have also published an open source bundle for this.

[Simple REST Adapter Bundle](https://github.com/w-vision/SimpleRESTAdapterBundle)
[CI HUB Adapter Bundle](https://github.com/w-vision/CIHubAdapterBundle)

Any feedback is appreciated, and I hope you guys enjoy!
